doc/ansible/dev/developer_guide.md

```diff
@@ -31,6 +31,11 @@ Python][openshift_restclient_python] package. This can be installed from pip:
 $ pip3 install openshift
 ```
 
+Finally, a user must install the Ansible Kubernetes collection from ansible-galaxy:
+```bash
+$ ansible-galaxy install community.kubernetes
```

Comment:
This would need to be `ansible-galaxy collection install community.kubernetes` (at least for now). Otherwise we can use a `requirements.yml` file and just have it be `ansible-galaxy install -r requirements.yml` (and that could have any possible versioning requirements and other collections added as needed...

Reply:
(I think I would prefer a `requirements.yml` file, even if it didn't have a version constraint, so it would be like:

```
---
collections:
  - name: community.kubernetes
```

Reply:
I always mess that up, good catch. As for the requirements.yml file, I'd also like to do it that way, but I think it's probably out of scope for this PR and should be done in a followup if we want to get it in